fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -250,12 +286,12 @@ const CreateElection = () => {
       {questions.map((value, idx) => (
```

Comment:
```suggestion
      {questions.map((question, idx) => (
```

?

Reply:
This was avoided to not have some `question.question` few lines below, I think. I am not sure if it would be clearer. Changed to a longer but better name.